src/olympia/blocklist/models.py

```diff
@@ -383,6 +387,12 @@ def save_to_blocks(self):
 
         self.update(signoff_state=self.SIGNOFF_PUBLISHED)
 
+    @classmethod
+    def get_submission_from_guid(cls, guid, states=SIGNOFF_STATES_FINISHED):
+        return (
```

Comment:
Will this return an empty list if nothing is found, or `None`?

Reply:
it returns a queryset which is falsey when there are no results.